lib/src/components/isometric_tile_map_component.dart

```diff
@@ -33,7 +33,14 @@ class IsometricTileMapComponent extends PositionComponent {
   /// Optionally provide a new tile size to render it scaled.
   Vector2 destTileSize;
 
-  IsometricTileMapComponent(this.tileset, this.matrix, {this.destTileSize});
+  IsometricTileMapComponent(
+    this.tileset,
+    this.matrix, {
+    this.destTileSize,
```

Comment:
size doesn't quite make sense here because destTileSize + map size is used. I should probably override the size so it returns the real size of the resulting render but that is complicated because of the transformed coordinate system. might try that later.